static/src/javascripts/projects/common/modules/commercial/dfp/dfp-obj.js

```diff
@@ -0,0 +1,4 @@
+define(function () {
```

Comment:
I'm guessing this deliberately has no dependencies, but I'm not sure why? The reset function in `dfp-api`, and the decomposed function modules all extend `dfp-obj`, why is this not done here? Is is possible for a client to require `dfp-obj` and attempt to invoke a function on an empty object?